pkg/controllers/namespace/namespace_controller.go

```diff
@@ -247,7 +239,7 @@ func (r *NamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			}
 		}
 
-		if _, err := r.createVPCCR(&ctx, obj, ns, ncName, createVpcName); err != nil {
+		if _, err := r.createNetworkInfoCR(&ctx, obj, ns, ncName); err != nil {
 			return common.ResultRequeueAfter10sec, nil
 		}
 		if err := r.createDefaultSubnetSet(ns); err != nil {
```

Comment:
just leave a question here, do we need to create default subnetset cr for shared vpc here

Reply:
as discussed